civihr_employee_portal/src/Webform/LocationTypeIDConvertor.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\Webform;
+
+class LocationTypeIDConvertor implements WebformTransferConvertor {
+
+  const MAPPING_KEY = 'locationTypes';
+
+  /**
+   * @inheritdoc
+   */
+  public static function preExport(\stdClass $node) {
+    $locationTypeIDs = self::getLocationTypeIDsByRef($node);
+    $params = ['id' => ['IN' => $locationTypeIDs]];
+    $locationTypes = civicrm_api3('LocationType', 'get', $params)['values'];
+    $locationTypes = array_column($locationTypes, 'name', 'id');
+    $node->customMapping[self::MAPPING_KEY] = $locationTypes;
+  }
+
+  /**
+   * @inheritdoc
+   */
+  public static function preImport(\stdClass $node) {
```

Comment:
Would be good to add a description for the preImport and preExport functions rather than they inheriting from the generic doc.